File: chapter5/mario.py
from cs50 import get_int
while True:
    # prompts user for proper input
    num = get_int("Height: \n")
    if(num >= 1 and num < 9):
        break

for i in range(num):
    # prints spaces before the hashes
    for j in range(num - (i + 1)):
        print(" ", end='')
    # prints out hashes
    for j in range(i + 1):
        print("#", end='')
    # prints return after line
    print()

File: chapter1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int number;
    do
    {
        number = get_int("Height: \n"); // repeat the question until valid input
    }
    while (number <= 0 || number > 8);


    for (int j = 0; j < number; j++) // repeat for the rows
    {


        for (int i = 0; i < number - (j + 1); i++) //print spaces before #
        {

            printf(" ");
        }


        for (int i = 0; i < j + 1; i++) //print #s
        {
            printf("#");
        }



        printf("\n");
    }

    // printf("\n");

}